targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L412xB/TOOLCHAIN_ARM/startup_stm32l412xx.S

```diff
@@ -0,0 +1,328 @@
+;*******************************************************************************
+;* File Name          : startup_stm32l412xx.s
+;* Author             : MCD Application Team
+;* Description        : STM32L412xx Ultra Low Power devices vector table for MDK-ARM toolchain.
+;*                      This module performs:
+;*                      - Set the initial SP
+;*                      - Set the initial PC == Reset_Handler
+;*                      - Set the vector table entries with the exceptions ISR address
+;*                      - Branches to __main in the C library (which eventually
+;*                        calls main()).
+;*                      After Reset the Cortex-M4 processor is in Thread mode,
+;*                      priority is Privileged, and the Stack is set to Main.
+;*******************************************************************************
+;*
+;* <h2><center>&copy; Copyright (c) 2018 STMicroelectronics.
+;* All rights reserved.</center></h2>
+;*
+;* This software component is licensed by ST under Apache License, Version 2.0,
+;* the "License"; You may not use this file except in compliance with the
+;* License. You may obtain a copy of the License at:
+;*                        opensource.org/licenses/Apache-2.0
+;*
+;*******************************************************************************
+;* <<< Use Configuration Wizard in Context Menu >>>
```

Comment:
could we get here SPDX identifier as well (other files have it) ?

Reply:
No... none of .S ST files have SPDX, as they are copied from ST github repo

Reply:
Can we request it upstream, it would be useful for projects.